swagger.yaml

```diff
@@ -1661,4 +1920,107 @@ components:
         monthFlextimeBalance:
           type: number
           format: double
-          nullable: true
\ No newline at end of file
+          nullable: true
+
+    Memo:
+      type: object
+      properties:
+        id:
+          type: string
+          description: Unique identifier for the memo
+        title:
+          type: string
+          description: Title of the memo
+        content:
+          type: string
+          description: Content or description of the memo
+        fileId:
+          type: string
+          description: Google Drive pdf file ID associated with the memo
+        date:
+          type: string
+          format: date
+          description: Date the memo was created or relevant to the memo
+        summary:
+          type: string
+          description: Summary of the memo content
+        translatedContent:
+          type: string
+          description: Translated content of the memo
+        createdAt:
+          type: string
+          format: date-time
+          description: Creation timestamp
+        updatedAt:
+          type: string
+          format: date-time
+          description: Last update timestamp
+        tags:
+          type: array
+          items:
+            type: string
+          description: Tags associated with the memo
+        attachments:
+          type: array
+          items:
+            type: string
+          description: List of attachments related to the memo
+        assignedTo:
+          type: array
+          items:
+            type: string
+          description: List of users assigned to this memo
+
+    Card:
```

Comment:
Consider whether these schemas should be named more specifically e.g. googleDriveMemo, TrelloCard, TrelloMember. It's your call